vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -155,23 +155,30 @@ def _get_page_divs(self, page: Page) -> _PageDivsType:
         return html.Div([dashboard_title, settings, page_title, nav_bar, nav_panel, control_panel, components])
 
     def _arrange_page_divs(self, page_divs: _PageDivsType):
-        left_header_divs = [page_divs["dashboard-title"]]
+        page_header_divs = [page_divs["dashboard-title"]]
         left_sidebar_divs = [page_divs["nav-bar"]]
         left_main_divs = [
-            html.Div(left_header_divs, id="left-header", hidden=_all_hidden(left_header_divs)),
             page_divs["nav-panel"],
             page_divs["control-panel"],
         ]
+        right_header_divs = [page_divs["page-title"]]
+
+        # Apply different container position logic based on condition
+        page_header_divs.append(page_divs["settings"]) if self.title else right_header_divs.append(
```

Comment:
In the future this would be if there was a logo but no title?

Reply:
Yes, exactly. Here is actually a previous commit where I've prototyped with the logo: https://github.com/mckinsey/vizro/pull/238/commits/ac696cf4bad24e9c77324bd7fbaca44ad277c3c5

I've removed it from this PR as this will be part of another PR

Reply:
Separate PR on the logo: https://github.com/mckinsey/vizro/pull/248

Reply:
It's not very good practice to use a ternary operator for something that doesn't have an explicit return value but instead relies on side-effects (i.e. `append` here), Better to spell out in full as:
```
if self.title:
    page_header_divs.append
else:
   ...
```